examples/lode_linear/environement.yml

```diff
@@ -0,0 +1,15 @@
+name: lode_linear
+dependencies:
+  - python=3.11
+  - pip
+  - pip:
+    # sphinx-gallery and dependencies
+    - sphinx
+    - pillow
+    - sphinx-gallery
```

Comment:
Since these are used for every example, can't we put them in the tox file?

Reply:
We could install them from inside `tox.ini` yes